### ISSUE TITLE
app path in windows

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#439]
Apparently the app path is being set wrong in windows to `"app/*": "lib\\app/*.js",`.

**Comment by Alxandr** [comment]
This is given that it's set in the `package.json` to `lib/app`. The path gets normalized to what the OS uses I suspect. Should probably just be a simple `replace(/\\/g, '/')` somewhere.

**Comment by Alxandr** [comment]
Same error happens when I try to do `bundle --inject`. The bundle-path shows up as `lib\\app`

**Comment by u9520107** [comment]
@guybedford I just tested on windows with jspm 0.11.5, the bundling injection still uses '\\' instead of '/'.

**Comment by guybedford** [comment]
@u9520107 thanks for the update - did you try with `jspm install -f`?

**Comment by u9520107** [comment]
install -f effects bundling injection path?
Anyway I did run jspm install -f in my project folder and still the records goes like:
```
"bundles": {
  "bundle\\base": [...]
}
```
Even the message output by the bundler says "bundle\base" was added to config.js instead of "bundle/base".
Doesn't really matter much if config.js isn't not commit with the project I guess. But if it is, then there will be "bundle/base" entry created by unix based systems, and "bundle\base" entry created by windows.
